src/Http/Requests/VerifySMSToken.php

```diff
@@ -28,7 +28,7 @@ public function authorize()
     public function rules()
     {
         return [
-            'token' => 'required|string|numeric',
+            'token' => ['required', 'string', 'regex:/^\d$/'],
```

Comment:
This regex doesn't work. This does though:

```suggestion
            'token' => ['required', 'string', 'regex:/^\d+$/'],
```

Reply:
Thanks!